data/campaigns/Under_the_Burning_Suns/scenarios/02_Across_the_Harsh_Sands.cfg

```diff
@@ -840,26 +840,17 @@
             [/have_unit]
         [/filter_condition]
 
-        [store_unit]
-            variable=bandits
-            kill=yes
+        [modify_unit]
             [filter]
                 role="Black Lieutenant"
                 [or]
                     role="Black Hand Bandit"
                 [/or]
             [/filter]
-        [/store_unit]
-        [foreach]
-            array=bandits
-            [do]
-                {CLEAR_VARIABLE this_item.ai_special,this_item.status.guardian}
-                [unstore_unit]
-                    variable=this_item
-                [/unstore_unit]
-            [/do]
-        [/foreach]
-        {CLEAR_VARIABLE bandits}
+            [status]
+                guardian=no
+            [/status]
+        [/modify_unit]
```

Comment:
This one is still worth doing, but needs testing. The old code changes both `ai_special` and `status.guardian`, why does the new code only change `status.guardian`?

Reply:
`ai_special=guardian` is a very old feature that was replaced with the `guardian` status and does nothing but set it to yes. Setting `ai_special` to anything other than `guardian` has no effect.